packages/ag-charts-community/src/chart/interaction/zoomManager.ts

```diff
@@ -118,8 +124,8 @@ export class ZoomManager extends BaseManager<'zoom-change', ZoomChangeEvent> {
 class AxisZoomManager {
     private axis: ChartAxisLike;
     private currentZoom: ZoomState;
-    private pendingZoom: ZoomState | null = null;
     private readonly initialZoom: ZoomState;
+    private state = new StateTracker<ZoomState>();
```

Comment:
nit: if initialised this inside the constructor, you can remove `initialZoom` by providing the value to as the default state value.
```
// replace:
this.initialZoom = this.currentZoom = { min, max };
// with:
this.state = new StateTracker({ min, max });
this.currentZoom = this.state.stateValue();
```

Reply:
Oh of course, thanks!